### ISSUE TITLE
Copy or Move list

### ISSUE THREAD

**Issue by morphos88** [RestyaPlatform/board#1283]
Add possibility like card to copy or move list in other tab

**Comment by lmarikannan** [comment]
@morphos88 Currently you can move list to other board. You need functionality to copy list to other board. Is it right?

**Comment by morphos88** [comment]
Yes it's right actually we copy list in board and move to other board.
